lib/net.js

```diff
@@ -935,7 +936,7 @@ function internalConnect(
       localAddress = localAddress || '::';
       err = self._handle.bind6(localAddress, localPort);
     } else {
-      self.destroy(new TypeError('Invalid addressType: ' + addressType));
+      self.destroy(new ERR_INVALID_ADDRESS_FAMILY(addressType));
```

Comment:
I think this is unreachable now, so I removed the test. I'm going to replace it by a CHECK when https://github.com/nodejs/node/issues/19503 is fixed.